deeppavlov/models/morpho_tagger/cells.py

```diff
@@ -12,24 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import keras.activations as kact
-import keras.backend as kb
-import keras.initializers as kinit
-import keras.layers as kl
 import numpy as np
-from keras.engine.topology import InputSpec
+import tensorflow.keras.backend as K
+from tensorflow.keras import activations
```

Comment:
The name ``activations`` is lowercase and therefore ambigious, it can be easily used in the code with another meaning, switch back to the old notation with ``tensorflow.keras`` instead of ``keras``.

Reply:
refactored